core/controllers/domain_objects_validator.py

```diff
@@ -96,3 +98,21 @@ def validate_collection_change(obj):
     # No explicit call to validate_dict method is necessary, because
     # CollectionChange calls validate method while initialization.
     collection_domain.CollectionChange(obj)
+
+
+def validate_email_dashboard_data(data):
+    # type: (Dict[String, Any]) -> None
```

Comment:
Please try to get rid of the `Any` here

Reply:
From the example you send it can be `Optional[Union[bool, int]]`.

Reply:
Thank you I am new to this.
I have looked into [constants.EMAIL_DASHBOARD_PREDICATE_DEFINITION](https://github.com/oppia/oppia/blob/7a412b3da5fa317581452c9517310e6f1884f363/assets/constants.ts#L5847)
the all values of the dicts are either boolean or an integer. 
Thank you for help.